gli/core/gl.inl

```diff
@@ -146,8 +146,6 @@ namespace gli
 			{INTERNAL_D24S8, EXTERNAL_DEPTH_STENCIL, TYPE_NONE},					//FORMAT_D24_UNORM_S8_UINT,
 			{INTERNAL_D32FS8X24, EXTERNAL_DEPTH_STENCIL, TYPE_NONE},				//FORMAT_D32_SFLOAT_S8_UINT,
 
-			{INTERNAL_RGB_DXT1, EXTERNAL_NONE, TYPE_NONE},							//FORMAT_RGB_DXT1_UNORM,
```

Comment:
OpenGL does, but DirectX doesn't. That's the problem, there is nothing in
the header that specifies an alpha channel.